lib/config.sh

```diff
@@ -1,8 +1,8 @@
 #! /bin/bash
 
 function verify-not-empty {
-	if [[ "${!1}" == "" ]]; then
-		echo "Variable $1 is not set.  Please set the value on the command line, enironment, or in gcloud config." && exit 1
+	if [[ -z "$1" ]]; then
```

Comment:
This will only verify that the function received a non-null string, right?  Not weather the variable name passed to it has been assigned a value.  e.g. L11
`verify-not-empty "GCP_EMAIL"`

Reply:
True. OK, I'll revert that change.